frontend/src/lang/modules/tw.ts

```diff
@@ -1708,6 +1708,10 @@ const message = {
         update: '更新',
         versionSelect: '請選擇版本',
         operatorHelper: '將對選中應用進行 {0} 操作，是否繼續？',
+        startOperatorHelper: '將對選中應用進行 啟動 操作，是否繼續？',
+        stopOperatorHelper: '將對選中應用進行 停止 操作，是否繼續？',
+        restartOperatorHelper: '將對選中應用進行 重啟 操作，是否繼續？',
+        reloadOperatorHelper: '將對選中應用進行 重載 操作，是否繼續？',
         checkInstalledWarn: '未檢測到 {0} ,請進入應用商店點擊安裝！',
         gotoInstalled: '去安裝',
         search: '搜索',
```

Comment:
The provided code snippet seems to be incomplete as it lacks further context on its purpose and function. Could you please share more details or clarify which portion of the text within this snippet needs checking?